rest.yaml

```
@@ -249,12 +249,927 @@ paths:
+              schema:
+                  $ref: "#/components/schemas/Club"
+
+  /clubs/{clubId}/teams:
+    get:
+      summary: "get List of Teams by ClubId"
+      operationId: getListOfTeamsByClubId
+      parameters:
+        - name: "clubId"
+          in: "path"
+          required: true
+          schema: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)